fastmri/data/mri_data.py

```diff
@@ -358,3 +360,111 @@ def __getitem__(self, i: int):
             sample = self.transform(kspace, mask, target, attrs, fname.name, dataslice)
 
         return sample
+
+
+class AnnotatedSliceDataset(SliceDataset):
+    def __init__(
+            self,
+            root: Union[str, Path, os.PathLike],
+            challenge: str,
+            transform: Optional[Callable] = None,
+            use_dataset_cache: bool = False,
+            sample_rate: Optional[float] = None,
+            volume_sample_rate: Optional[float] = None,
+            dataset_cache_file: Union[str, Path, os.PathLike] = "dataset_cache.pkl",
+            num_cols: Optional[Tuple[int]] = None,
+            use_annotation: bool = False,
```

Comment:
Why do we have this flag for an `AnnotatedSliceDataset`?

Reply:
Yes, it's unnecessary in the subclass. I'll delete it.